cachito/web/models.py

```diff
@@ -376,6 +377,7 @@ def to_json(self, verbose=True):
             env_vars_json[env_var.name] = env_var.value
         rv = {
             "id": self.id,
+            "created": self.created.isoformat(),
```

Comment:
What is timezone here? is that UTC?

Reply:
isn't this deployment detail, that shouldn't be even considered upstream?

What if I change pod timezone, or run this as VM?

Reply:
I think I didn't get your point here :sweat_smile:. Are you suggesting we pin the timezone in the application level?

Reply:
```python
>>>datetime.datetime.utcnow().isoformat()                                                                                                                                                                                                 
'2021-10-06T17:17:40.127838'
```
it provides UTC time whatever is local timezone

Reply:
as I was informed, cachito works with local timezone settings on other places too, so moving this to UTC here, would cause more problems than solutions